hack/cluster-deploy.sh

```diff
@@ -21,7 +21,8 @@ set -ex pipefail
 
 DOCKER_TAG=${DOCKER_TAG:-devel}
 KUBEVIRT_DEPLOY_CDI=${KUBEVIRT_DEPLOY_CDI:-true}
-CDI_DV_GC=${CDI_DV_GC:-0}
+CDI_DV_GC_DEFAULT=0
```

Comment:
this will get changed after https://github.com/kubevirt/containerized-data-importer/pull/2754 merges?

Reply:
Yep, `CDI_DV_GC_DEFAULT` will change to -1, so we won’t patch CDIConfig here unless explicitly asked by the env var. However, unfortunately some KubeVirt func tests currently need to know what the GC default is (whether `CDIConfig.Spec.DataVolumeTTLSeconds` == nil means GC enabled/disabled), so once CDI new default is consumed by KubeVirt [this line](https://github.com/kubevirt/kubevirt/blob/main/tests/libstorage/datavolume.go#L175) also needs to be fixed. We once considered adding `CDIConfig.Status.DataVolumeTTLSeconds` reflecting the state also when it's default (-1/0) so KubeVirt won't need to have the knowledge, however we decided that it just complicates things.